submissions/MarharytaBoichenko/html_css_popup/sass/main.scss

```diff
@@ -0,0 +1,304 @@
+$primaryColor: #000;
+$mainFont: 'Merriweather', serif;
+$btnColor: #416aff;
+$bcgColor: #ffffff;
+
+html {
+  box-sizing: border-box;
+  min-height: 100%;
+}
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  min-height: 100%;
+  font-family: $mainFont;
+  font-size: 16px;
+  line-height: 1.71;
+  letter-spacing: 0.03em;
+  color: $primaryColor;
+  background-image: url('../popup-icons/ua.jpeg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+h1,
+h2,
+h3,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: currentColor;
```

Comment:
currentColor?

Reply:
isn`t it correct to use currentColor  to  remove blue  from all a  tag?

Reply:
I am just trying to understand what means `currentColor`. Which actual color is it? It looks like variable, but I can't find it

Reply:
it is a keyword , as I  understand it works  like  inheritance, for all a  tag  the color will be  the color of parent element,  if we do not write  another color for  links in code below
https://www.digitalocean.com/community/tutorials/css-currentcolor

Reply:
Nice, now I will know it 🚀 Thanks :)